bootstrap/sql/com.mysql.cj.jdbc.Driver/v007__create_db_connection_info.sql

```diff
@@ -36,3 +36,7 @@ CREATE TABLE IF NOT EXISTS alert_action_def (
     PRIMARY KEY (id),
     UNIQUE (name)
 );
+
+UPDATE pipeline_service_entity 
+SET json = JSON_REMOVE(json, '$.connection.config.configSource')
+WHERE serviceType = 'Dagster';  
```

Comment:
should we add the token for the services that we are removing it?

Reply:
otherwise users will need to reenter it

Reply:
done. Thanks